R/type2.R

```diff
@@ -32,20 +32,23 @@
 #' @keywords internal
 #' @param x,y Either vector types; i.e.
 #' @export
-vec_type2 <- function(x, y) {
-  return(.Call(vctrs_type2, x, y))
+vec_type2 <- function(x, y, ..., x_arg = "", y_arg = "") {
```

Comment:
Are you sure we should use `...` here? It saves a bunch of typing, but is it the right thing to do?

Reply:
If we don't add `...` to the generic, it'll be hard to add new arguments once vctrs is more widely used. Shouldn't we add `...` to all generics?

Reply:
I'm not sure. Maybe it's ok now that we have ellipsis?